### PR TITLE
change deeplink url to betwixt.space

### PR DESCRIPTION
**Pull request by danielraad**
- this is a simple change to use betwixt.space
- I am still debugging as to why the link is not clickable on Android

### CODE REVIEW

File: src/pages/space/space.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export class SpacePage {
     params: NavParams
   ) {
     this.uid = params.get('uid');
-    this.spaceLink = `betwixt://betwixt.com/?group_uid=${this.uid}`;
+    this.spaceLink = `betwixt://betwixt.space/?group_uid=${this.uid}`;
```
Comment (arodr967) [inline src/pages/space/space.ts]: Are you sure there aren't any other changes you need to make?
I think @Palacios95 had mentioned that there was something else that needed to be changed...

**Review by Palacios95 — COMMENTED** [review]
We need to test this on an android phone if we haven't already. @arodr967 could push this to the phonegap build and I'll use and android phone to test it.

File: src/pages/space/space.ts (hunk 1 above)
Comment (Palacios95) [reply src/pages/space/space.ts]: I believe that the "extra" thing we needed was for iOS, but that was a while back and I think it's resolved already.

**Review by arodr967 — APPROVED** [review]
Awesome!

**Pull request merged**